packages/toolkit/src/dao/acl/getTransactionPath.ts

```diff
@@ -24,28 +21,14 @@ export async function getTransactionPath(
   appAddress: string,
   method: string,
   params: any[],
+  account: string,
   environment: string
 ): Promise<TransactionPath> {
-  const wrapper = await initWrapper(dao, environment)
+  const { chainId } = useEnvironment(environment)
```

Comment:
The `chainId` is one of the values that we may need to initialize Aragon Connect. Maybe we can get it using `use-waller`?